proposals/0000-add-support-for-css-color-function.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Add support for CSS color() function
+author:
+  - Ryan Linton
+  - Yulian Glukhenko
+date: 2023-11-20
+---
+
+# RFC0000: Add support for CSS color() function
+
+## Summary
+
+React Native does [not currently support](https://github.com/facebook/react-native/issues/41517) wide color gamut color spaces (i.e. display-p3). This proposal discusses adding support for the CSS color() function enabling support for additional wide-gamut color spaces.
```

Comment:
out of curiosity from a non-english first-language speaker: is it more correct _wide gamut color_ or _wide color gamut_? 🤔 

From a stranger perspective, _wide_ is attribute of _gamut_, so I would spell that as _wide gamut_ colors, but I might be completely wrong here! 😅

```suggestion
React Native does [not currently support](https://github.com/facebook/react-native/issues/41517) wide gamut color spaces (i.e. display-p3). This proposal discusses adding support for this color space in React Native, covering:
- JS implementation
- platform changes
```

Reply:
I believe you're correct that _wide gamut color_ is ___more___ correct than _wide color gamut_. I've seen it used pretty interchangeably but I suppose  _wide gamut color_ would be more correct if you're referring to a color while _wide color gamut_ would still make sense if you're referring to the gamut.